Building-ChakraCore.md

```diff
@@ -45,6 +45,26 @@ Alternatively, you can build ChakraCore without MSVC runtime dependency. Pass an
 msbuild ... /p:RuntimeLib=static_library ...
 ```
 
+## Windows + ICU (Experimental) ##
```

Comment:
nit - may want to consider an extra # as this sounds like a section under ## Windows ##. Same goes fro its subsections.